src/test/java/org/ossgang/commons/utils/GeneralDecimalFormatTest.java

```diff
@@ -0,0 +1,56 @@
+package org.ossgang.commons.utils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GeneralDecimalFormatTest {
+
+    private GeneralDecimalFormat format = new GeneralDecimalFormat(6, 4);
```

Comment:
Consider Testing the other public constructors ... ? Especially the one taking a string as first argument ....